doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+---
+* __[Goal](#goal)__
+* __[Do I have to transition?](#do-i-have-to-transition-to-redux)__
+* __[A Simple Example](#a-simple-example)__
+    * [A Basic Flux App](#a-basic-flux-app)
+    * [The Redux Version](#the-redux-version)
+    * [Basic Example: The Main Differences](#basic-example:-the-main-differences)
+    * [Basic Conversion Step by Step](#basic-conversion-step-by-step)
+* __[An Advanced Example](#an-advanced-example)__
+    * [An Advanced Flux App](#an-advanced-flux-app)
+    * [Converted to Redux](#converted-to-redux)
+    * [Advanced Conversion Step by Step](#advanced-conversion-step-by-step)
+    * [What if the Advanced Conversion is too Daunting?](#what-if-the-advanced-conversion-is-too-daunting?)
+* __[Influx Architecture](#influx-architecture)__
+    * [What is Influx?](#what-is-influx?)
+    * [Should You Use It](#should-you-implement-an-influx-architecture-during-the-transition?)
+        * [Advantages](#advantages)
+        * [Disadvantes](#disadvantages)
+    * [Important Terms](#important-terms)
+    * [Steps to an Influx Refactor](#steps-to-an-influx-refactor)
+        * [Phase 1: Get Ready to Refactor](#phase-1:-get-ready-to-refactor)
+        * [Phase 2: Update Stores](#phase-2:-incrementally-update-stores)
+        * [Phase 3: Influx to Redux](#phase-3:-influx-to-redux)
+
+## Goal
+The goal of this document is explain major elements of transitioning from Flux to Redux. This includes explanation of both a simple and more advanced conversion, and the introduction of a new architecture as a last resort for the most extreme cases.
+
+If, through this process, the document or examples miss any edge cases you encounter, please create an issue or reach out so it can be added.
+
+## Do I have to transition to Redux?
+No. OverReact Redux is meant to help to provide a recommended state management method for OverReact components, as well as provide benefits over w_flux and built_redux. Those benefits may provide enough reason to make a refactor worthwhile. If, after evaluating the benefits and effort, the juice doesn't seem worth the squeeze, then no need to worry about it!
+
+To evaluate if the refactor is worth it, the details of OverReact Redux can be found in the [OverReact Redux doc](./over_react_redux_documentation.md). That document paired with this guide will illustrate the full scope of costs and benefits.
+
+## A Simple Example
+To illustrate the basic changes that will occur, this section will go through a basic Flux architecture and then show that same system with Redux instead. This section will also have a step by step list of instructions for an update as straightforward as this one. An actual working example can be found in the [Flux to Redux Example](../web/flux_to_redux/simple/readme.md).
```

Comment:
This was originally what I wanted to do, but it seems like because that path doesn't have a specific endpoint, trying to use the link doesn't do anything. I actually created the readme just to add that endpoint so people could click on it. Should that path navigate me to the directory? I may be overlooking something

Reply:
@joebingham-wk the link you've got now doesn't work at all (files on github are case sensitive).

Changing it to `../web/flux_to_redux/simple` navigates me correctly to https://github.com/Workiva/over_react/tree/0731dbb71e883a13670efb54297681d3e60230a1/web/flux_to_redux/simple

Reply:
Ohh that makes more sense, @aaronlademann-wf. I had always used the IDE markdown previewer to view markdown docs (and do my link testing), so I wasn't even thinking of Github 🤦‍♂ Thank you for pointing that out - I updated references to point to the simple / advanced directory instead of the readme files